lib/constants/index.ts

```diff
@@ -56,7 +56,7 @@ export const addresses = {
 };
 
 export const urls = {
-  home: "https://www.klimadao.finance/",
+  home: "/",
```

Comment:
we need the hardcoded url otherwise when used in the dapp, this will just take you to the dapp root rather than the landing page.
if you want root just inline `"/"`